.ci/prepare-settings.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+mkdir -p .ci-temp
+cp .ci/release-settings.xml .ci-temp/
+
+checkForVariable() {
+  if [[ "$(declare -p "$1")" =~ ^declare\ -x ]]; then
```

Comment:
why not simple `[ -v $VAR ]` ?

https://www.cyberciti.biz/faq/see-check-if-bash-variable-defined-in-script-on-linux-unix-macos/

Reply:
```
✘-1 ~/java/github/romani/checkstyle [Rahulkhinchi03/setting L|✚ 1] 
$ export SONATYPE_USER="usssr" && ./.ci/prepare-settings.sh
Error: Define SONATYPE_PWD environment variable
✘-1 ~/java/github/romani/checkstyle [Rahulkhinchi03/setting L|✚ 1] 
 $ 
✘-1 ~/java/github/romani/checkstyle [Rahulkhinchi03/setting L|✚ 1] 
19:51 $ git diff
diff --git a/.ci/prepare-settings.sh b/.ci/prepare-settings.sh
index 1012522..e43eac4 100755
--- a/.ci/prepare-settings.sh
+++ b/.ci/prepare-settings.sh
@@ -5,17 +5,22 @@ mkdir -p .ci-temp
 cp .ci/release-settings.xml .ci-temp/
 
 checkForVariable() {
-  if [[ "$(declare -p "$1")" =~ ^declare\ -x ]]; then
+  VAR_NAME=$1
+  if [ -v ${!VAR_NAME} ]; then
     echo "Error: Define $1 environment variable"
     exit 1
   fi
 }
```

Reply:
Changes Done